src/commands/flutter-symbols/upload.ts

```diff
@@ -194,7 +194,7 @@ export class UploadCommand extends Command {
 
   private async getGitMetadata(): Promise<RepositoryData | undefined> {
     try {
-      return await getRepositoryData(await newSimpleGit(), this.repositoryUrl)
+      return await getRepositoryData(await newSimpleGit(), this.repositoryUrl, undefined)
```

Comment:
nitpick: couldn't we have an optional parameter to the function instead of passing `undefined`?